packages/std/src/testing/message_info.rs

```diff
@@ -0,0 +1,83 @@
+use crate::{Addr, Coin, MessageInfo};
+
+/// A constructor function for [`MessageInfo`].
+///
+/// This is optimized for writing testing contract. It
```

Comment:
This sentence sounds weird to me. Maybe you meant "This is designed for writing contract tests" or something like that?

Reply:
Right, thanks, fixed